### PR TITLE
Re-built the Booking Modal as plain CSS div container

### PR DESCRIPTION
**Pull request by scottrumsby**
Due to issues with IE 11 and the document.activeElement call, Bootstrap Modals were not working in this situation so re-wrote the component as a div box styled to match bootstrap with the necessary logic to replace the modal hooks.  also fixed issues with selection of invigilators and added logic to auto focus on the text input when opening.

### CODE REVIEW

File: frontend/src/store/index.js [hunk 1]
```diff
@@ -309,6 +309,20 @@ export const store = new Vuex.Store({
       return false
     },
 
+    is_liaison_designate(state) {
```
Comment (sjrumsby) [inline frontend/src/store/index.js]: Why are these here? Are they not already defined like 10 lines up?

File: frontend/src/store/index.js [hunk 2]
```diff
@@ -309,6 +309,20 @@ export const store = new Vuex.Store({
...
+      if(state.user.liaison_designate){
+        return true
+      }
+      return false
+    },
+
+    is_ita_designate(state) {
```
Comment (sjrumsby) [inline frontend/src/store/index.js]: Same goes for this one being defined up a bit.

File: frontend/src/booking/booking-modal.vue [hunk 3]
```diff
@@ -315,11 +347,27 @@
...
+        console.log(this.invigilatorId)
+        console.log(this.clickedRow)
+        if (!this.invigilatorId && this.clickedRow) {
+          console.log('we did it')
+          this.clickedRow._rowVariant = ''
+        }
+      },
       rowClicked(item) {
-        this.selectedInvigilator = item.invigilator_id
+        console.log(item)
```
Comment (sjrumsby) [inline frontend/src/booking/booking-modal.vue]: Console log

File: frontend/src/booking/booking-modal.vue [hunk 4]
```diff
@@ -175,15 +201,17 @@
         return 'Submit'
       },
       displayData() {
+        let m = this.exam.exam_type.number_of_minutes ? (this.exam.exam_type.number_of_minutes / 60) : 0
+        let l = `${this.exam.exam_type.number_of_hours}.${m} hrs`
```
Comment (sjrumsby) [inline frontend/src/booking/booking-modal.vue]: What's `l`? We can do better at naming variables than that!

File: frontend/src/booking/booking-modal.vue [hunk 5]
```diff
@@ -175,15 +201,17 @@
         return 'Submit'
       },
       displayData() {
+        let m = this.exam.exam_type.number_of_minutes ? (this.exam.exam_type.number_of_minutes / 60) : 0
```
Comment (sjrumsby) [inline frontend/src/booking/booking-modal.vue]: What's `m`, we can do better at naming variables than that!

File: frontend/src/store/index.js [hunk 6]
```diff
@@ -749,8 +763,8 @@ export const store = new Vuex.Store({
           context.commit('setUser', resp.data.csr)
           let officeType = resp.data.csr.office.sb.sb_type
           context.commit('setOffice', officeType)
-          context.commit('setDefaultCounter', resp.data.csr.office.counters.filter(
-            c => c.counter_name === DEFAULT_COUNTER_NAME)[0])
+         // context.commit('setDefaultCounter', resp.data.csr.office.counters.filter(
```
Comment (sjrumsby) [inline frontend/src/store/index.js]: Are these the things that are broken from the other changes that were merged? Have you tested them with the latest merge into `sprint-10-4`? Since all the manage.py bootstrap stuff now works, I would hope that this does now too.

File: frontend/src/store/index.js (hunk 1 above)
Comment (scottrumsby) [reply frontend/src/store/index.js]: merge conflict resolution gone awry...

File: frontend/src/store/index.js (hunk 2 above)
Comment (scottrumsby) [reply frontend/src/store/index.js]: see previous comment

File: frontend/src/booking/booking-modal.vue (hunk 3 above)
Comment (scottrumsby) [reply frontend/src/booking/booking-modal.vue]: i'd like to someday submit a PR free extraneous console.logs.  someday...
